### PR TITLE
Refactor repo structure to accomodate potential future activities and triggers

### PR DESCRIPTION
**Pull request by wcn00**
As per the meeting on Aug 24.  Rename activity to activity/publish and trigger to trigger/subscriber
I think that the connector can be left and referenced as is.

### CODE REVIEW

**Comment by flogo-oss** [comment]
[CLA assistant check](https://flogo.cla.cloud.tibco.com:443/project-flogo/messaging-contrib?pullRequest=9) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://flogo.cla.cloud.tibco.com:443/project-flogo/messaging-contrib?pullRequest=9) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://flogo.cla.cloud.tibco.com:443/check/project-flogo/messaging-contrib?pullRequest=9) it.</sub>

**Comment by lixingwang** [comment]
@wcn00 I think we need move function under trigger as well

**Comment by wcn00** [comment]
Will do with next pr containing runtime changes.

**Review by mellistibco — APPROVED** [review]

File: pulsar/go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/project-flogo/messaging-contrib/pulsar
+module github.com/wcn00/messaging-contrib/pulsar
```
Comment (mellistibco) [inline pulsar/go.mod]: This should be changed back to project-flogo

**Comment by wcn00** [comment]
I did revert that, but didn't get it into the pr (cause I don't do many 
pr's)  I'll get it in the next pr.

wcn

File: pulsar/go.mod (hunk 1 above)
Comment (fm-tibco) [reply pulsar/go.mod]: We should fix this in this PR.

**Review by mellistibco — CHANGES_REQUESTED** [review]
The module name should be reverted in this PR...

**Comment by wcn00** [comment]
I pushed the change.  I am not familiar with working with pr's in github.  Let me know if this is the wrong thing to have done.

**Review by mellistibco — APPROVED** [review]

**Review by vijaynalawade — COMMENTED** [review]
@wcn00 : Make sure refs in README.md are updated too.

**Review by fm-tibco — APPROVED** [review]

**Review by vijaynalawade — APPROVED** [review]

**Pull request merged**